### ISSUE TITLE
Lower CPU overhead of optimizer.step (MXFP8)

### ISSUE THREAD

**Issue by erhoo82** [NVIDIA-NeMo/Megatron-Bridge#1793]
MXFP8 optimizer step function involves high CPU overhead from (1) applying quantization, gradient zeroing, and optimizer function to an individual parameter, and (2) lots of tensor slicing.

We would need to use (1) a CUDA graph and (2) multi-tensor operations to lower CPU overhead.

**Comment by yaoyu-33** [comment]
Closing as part of issue cleanup — this issue has been stale with no recent activity. If you still need help, please open a new issue with updated details. Thank you!
